### DISCUSSION TITLE
基于PaddleX3.0的农业玉米雄蕊精准定位系统模型训练报错'RT-DETR-H' is not a registered model name.

### DISCUSSION THREAD

**Discussion by XzxtX** [PaddlePaddle/PaddleX#2384] [Q&A] [answered]
'RT-DETR-H' is not a registered model name.
![屏幕截图 2024-11-04 174838](https://github.com/user-attachments/assets/34176124-498a-469d-85b8-3aa02c8df7b9)

**Comment by cuicheng01** [comment, marked as answer]
可以参考文档安装PaddleDetection插件，`paddlex --install PaddleDetection --platform gitee.com`

**Reply by XzxtX** [reply]
您好，这个问题我已经解决，但是安装了这个之后运行代码出现了这个问题
![Uploading 屏幕截图 2024-11-08 172357.png…]()

**Reply by XzxtX** [reply]
![屏幕截图 2024-11-08 172357](https://github.com/user-attachments/assets/95b78baf-65f2-4830-a4b8-446301ce94d8)

**Reply by cuicheng01** [reply]
版本没有对上哦，在beta版本和beta1的版本接口差异比较大，可以升级下paddlex

**Reply by XzxtX** [reply]
> 版本没有对上哦，在beta版本和beta1的版本接口差异比较大，可以升级下paddlex

问题已经解决了paddleinferenceoption在这个版本不适用

**Reply by Praneshkumar-R** [reply]
For me its getting the same error like mode not registered . Pls can u tell me how u have corrected it
